Formula/p/perl-xml-parser.rb

```diff
@@ -0,0 +1,22 @@
+class PerlXmlParser < Formula
+  desc "Perl module for parsing XML documents"
+  homepage "https://github.com/toddr/XML-Parser"
+  url "https://cpan.metacpan.org/authors/id/T/TO/TODDR/XML-Parser-2.46.tar.gz"
+  sha256 "d331332491c51cccfb4cb94ffc44f9cd73378e618498d4a37df9e043661c515d"
+  license any_of: ["Artistic-1.0-Perl", "GPL-1.0-or-later"]
+  head "https://github.com/toddr/XML-Parser.git", branch: "master"
+
+  uses_from_macos "perl"
+
+  def install
+    ENV.prepend_create_path "PERL5LIB", libexec/"lib/perl5"
+    system "perl", "Makefile.PL", "INSTALL_BASE=#{libexec}"
+    system "make", "PERL5LIB=#{ENV["PERL5LIB"]}"
```

Comment:
Could we perhaps install directly into `prefix` rather than `libexec` so we don't need `PERL5LIB` everywhere?

Reply:
Hmm seems like our Perl might not read from `HOMEBREW_PREFIX/lib/perl5`. What we possibly might want to do is define `vendorlib` to that, and potentially move Perl's own core modules to be in libexec.

Reply:
@Bo98 maybe let's get this PR merged and followup with the file location stuff? :)

Reply:
I agree about this being able to wait until a a future PR.

Reply:
Sure, though you will have to go round and update every dependent to change `Formula["intltool"].opt_libexec` to `Formula["perl-xml-parser"].opt_libexec`.